doc/source/design.rst

```diff
@@ -357,7 +368,36 @@ example:
 Version Information When Upgrading
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--  Need a nice chart.
+Whenever a package needs an upgrade, if a ``--verbose-update`` is passed to
+Aura, then a detailed chart is produced, as described below.
+
+The coloured part is denoted with a ``<colour></colour>`` syntax.
```

Comment:
```... with ``<colour></colour>`` tags```, i.e. leave out the `a`.
Additionally, should we give an example of how to use those colour tags here?